modules/navigation/nav_map.h

```diff
@@ -136,6 +136,13 @@ class NavMap : public NavRid {
 	HashMap<gd::EdgeKey, ConnectionPair, gd::EdgeKey> connection_pairs_map;
 	LocalVector<gd::Edge::Connection> free_edges;
 
+	struct {
+		SelfList<NavRegion>::List region_list;
+		SelfList<NavLink>::List link_list;
+		SelfList<NavAgent>::List agent_list;
+		SelfList<NavObstacle>::List obstacle_list;
```

Comment:
`_list` suffix is redundant IMO - w/o it it would be `regions`, `links`, `agents`, and `obstacles`